lib/mayaUsd/utils/util.cpp

```diff
@@ -2079,171 +2079,171 @@ void UsdMayaUtil::GetFilteredSelectionToExport(
     }
 }
 
-float UsdMayaUtil::ConvertMTimeUnitToFloat(const MTime::Unit& unit)
+double UsdMayaUtil::ConvertMTimeUnitToDouble(const MTime::Unit& unit)
 {
-    float ret = 0.f;
+    double ret = 0.0;
     switch (unit) {
     case MTime::k2FPS: {
-        ret = 2.f;
+        ret = 2.0;
     } break;
     case MTime::k3FPS: {
-        ret = 3.f;
+        ret = 3.0;
     } break;
     case MTime::k4FPS: {
-        ret = 4.f;
+        ret = 4.0;
     } break;
     case MTime::k5FPS: {
-        ret = 5.f;
+        ret = 5.0;
     } break;
     case MTime::k6FPS: {
-        ret = 6.f;
+        ret = 6.0;
     } break;
     case MTime::k8FPS: {
-        ret = 8.f;
+        ret = 8.0;
     } break;
     case MTime::k10FPS: {
-        ret = 10.f;
+        ret = 10.0;
     } break;
     case MTime::k12FPS: {
-        ret = 12.f;
+        ret = 12.0;
     } break;
     case MTime::k15FPS: {
-        ret = 15.f;
+        ret = 15.0;
     } break;
     case MTime::k16FPS: {
-        ret = 16.f;
+        ret = 16.0;
     } break;
     case MTime::k20FPS: {
-        ret = 20.f;
+        ret = 20.0;
     } break;
     case MTime::k23_976FPS: {
-        ret = 23.976f;
+        ret = (24.0 * 1000.0) / 1001.0;
     } break;
     case MTime::k24FPS: {
-        ret = 24.f;
+        ret = 24.0;
     } break;
     case MTime::k25FPS: {
-        ret = 25.f;
+        ret = 25.0;
     } break;
     case MTime::k29_97FPS: {
-        ret = 29.97f;
+        ret = (30.0 * 1000.0) / 1001.0;
     } break;
     case MTime::k29_97DF: {
-        ret = 29.97f;
+        ret = (30.0 * 1000.0) / 1001.0;
     } break;
     case MTime::k30FPS: {
-        ret = 30.f;
+        ret = 30.0;
     } break;
     case MTime::k40FPS: {
-        ret = 40.f;
+        ret = 40.0;
     } break;
     case MTime::k47_952FPS: {
-        ret = 47.952f;
+        ret = (48.0 * 1000.0) / 1001.0;
     } break;
     case MTime::k48FPS: {
-        ret = 48.f;
+        ret = 48.0;
     } break;
     case MTime::k50FPS: {
-        ret = 50.f;
+        ret = 50.0;
     } break;
     case MTime::k59_94FPS: {
-        ret = 59.94f;
+        ret = (60.0 * 1000.0) / 100;
```

Comment:
Hey @dgovil! Sorry, only just noticing this as I'm integrating these changes internally.

I think `1001.0` was probably what was intended here rather than `100`?

Reply:
oh yes! I must have hit backspace. let me put up a new PR

Reply:
https://github.com/Autodesk/maya-usd/pull/1010